File "./typechecking/invalid/tunder_5.sk", line 6, characters 3-7:
The value returned from the function is incompatible with the return type '_[1]'
4 |
5 | fun bar(f: Int -> _): _ {
6 |   f(42)
  |   ^^^^^
7 | }

File "./typechecking/invalid/tunder_5.sk", line 5, characters 19-19:
The type: _[0]
3 | }
4 |
5 | fun bar(f: Int -> _): _ {
  |                   ^
6 |   f(42)

File "./typechecking/invalid/tunder_5.sk", line 5, characters 23-23:
Is not a subtype of: _[1]
3 | }
4 |
5 | fun bar(f: Int -> _): _ {
  |                       ^
6 |   f(42)
File "./typechecking/invalid/tunder_5.sk", line 2, characters 3-3:
The value returned from the function is incompatible with the return type '_[1]'
1 | fun foo(x: _): _ {
2 |   x
  |   ^
3 | }

File "./typechecking/invalid/tunder_5.sk", line 1, characters 12-12:
The type: _[0]
1 | fun foo(x: _): _ {
  |            ^
2 |   x

File "./typechecking/invalid/tunder_5.sk", line 1, characters 16-16:
Is not a subtype of: _[1]
1 | fun foo(x: _): _ {
  |                ^
2 |   x
